rest/route/agent.go

```diff
@@ -996,7 +996,17 @@ func (h *startTaskHandler) Run(ctx context.Context) gimlet.Responder {
 		}
 
 		idleTimeStartAt := host.LastTaskCompletedTime
-		if idleTimeStartAt.IsZero() || idleTimeStartAt == utility.ZeroTime {
+		if utility.IsZeroTime(idleTimeStartAt) {
+			idleTimeStartAt = host.BillingStartTime
+		}
+		if utility.IsZeroTime(idleTimeStartAt) {
```

Comment:
I feel like doing this identical if depending on the state of idleTimeStartAt is a little awkward.

Reply:
Does this make sense as a method on host since it repeats, and doing so might avoid this slightly awkward double "if" statement?

Reply:
When I went to move it onto the host I noticed a very similar method already existed. I tried to consolidate the two.